tests/Fixer/FunctionNotation/PhpdocToReturnTypeFixerTest.php

```diff
@@ -370,6 +370,36 @@ function bar() {}
             '<?php /** @return int */ fn(): int => 1;',
             '<?php /** @return int */ fn() => 1;',
         ];
+
+        yield 'arrow function (static)' => [
```

Comment:
```suggestion
        yield 'arrow function (as statement ended by ;)' => [
```

Reply:
This change was unneeded. This is exactly the same code as `arrow function` test case (defined above), but with `static` keyword, so case's name was more accurate before.